metaflow/plugins/argo/argo_live_run.py

```diff
@@ -166,9 +164,9 @@ def _wait(self, wait_timeout) -> None:
                     f" minutes out of a possible {wait_timeout}"
                 )
             loop_counter += 1
-            time.sleep(5)
+            time.sleep(5 - ((time.time() - start_time) % 5))
```

Comment:
Can you explain what this does in comment?

Reply:
Will this logic result in sleeping for negative amount of time?

Reply:
It can't go negative because x % 5 will never be greater than 5.

I added this because it takes time to lookup status, which made the status appear slightly less frequently than one per minute. I added this comment '# each loop takes 5s' as an inline comment. Is this sufficient? 

I could also just make it 5s each time to avoid any confusion. It's not a big deal. On larger flows, you'd occasionally see something like:

> Time waited: 78 minutes out of a possible 720
> Time waited: 79 minutes out of a possible 720
> Time waited: 80 minutes out of a possible 720
> Time waited: 82 minutes out of a possible 720
> Time waited: 83 minutes out of a possible 720

Skipping from 80 to 82 is a little odd, but not a big deal.

What do you think I should do? My sense is that, to avoid confusion, I change it back to 5s every time, but I don't feel strongly about it.

Reply:
Oh I see what you are doing now. I think for this feature it's not important to be off for a few seconds. Constant 5 seconds wait is fine.